dcat/index.html

```diff
@@ -3387,6 +3387,225 @@ <h2>Resource life-cycle</h2>
+
+<p>There are no common rules and criteria across domains to decide when dataset series should be created and how they should be organized. The situation is similar to the one concerning versioning (see <a href="#dataset-versions"></a>), and, likewise, DCAT does not adopt any specific definition of dataset series, and when and how they should be created and organized. The purpose of this section is limited to providing guidance on how dataset series can be specified in DCAT.</p>
+
+
+<section id="dataset-series-specification">
+
+<h2>How to specify dataset series</h2>
+
+<p>Existing DCAT implementations adopt two main alternative approaches to specifying dataset series:</p>
+
+<ol>
+<li>The dataset series is typed as a <code>dcat:Dataset</code>, whereas its child datasets are typed as <code>dcat:Distribution</code>'s.</li>
+<li>Both the dataset series and its child datasets are typed as a <code>dcat:Dataset</code>'s, and the two are usually linked by using the [[DCTERMS]] properties <code>dct:hasPart</code> / <code>dct:isPartOf</code>.</li>
+</ol>
+
+<p>In both cases, the dataset series is sometimes soft-typed by using the [[DCTERMS]] property <code>dct:type</code> (e.g., this is the approach used in [[GeoDCAT-AP]], and adopted in [[DCAT-AP-IT]] and [[GeoDCAT-AP-IT]]).</p>
+
+<p>Compared with the second option, the first one may have the advantage of simplifying metadata management, and avoiding the creation of datasets having an almost equivalent intensional description (i.e., datasets having the same values for almost all their metadata elements). On the other hand, this approach reduces the ability of being discovered, as distribution metadata are not rich as datasets' ones. Moreover, using distributions may result cumbersome or unfeasible when the number of child datasets is too high.</p>
+
+<p>As stated in the note to <a href="#Class:Distribution"></a>, DCAT recommends, as the default approach, typing both dataset series and child datasets as <code>dct:Dataset</code>'s, and linking them by using properties <code>dct:hasPart</code> and/or <code>dct:isPartOf</code>, and possibly soft-typing the dataset series via property <code>dct:type</code>.</p>
+
+<aside class="ednote">
+<p>The creation of a specific class for dataset series is under discussion.</p>
+</aside>
+
+<p>The approach based on the use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.</p>
```

Comment:
I wonder if we should drop this line..  Having two ways of representing the same might hamper interoperability. As far as I understood, there are no many actual implementations using the "distribution" alternative .. and as pointed out by Makx this somehow breaks the  DCAT backbone.  I think it is correct to mention both, but we should try to not encourage using the dcat:distribution in this way.

Reply:
Otherwise, if not dropping the entire line we can rephrase
```suggestion
<p>The approach based on the use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, only if it addresses more effectively the requirements of a given application scenario.</p>
```

Reply:
Actually, to my knowledge, distributions is how people are addressing this - also because of lack of support in catalogue platforms for dataset series. 

About replacing "whenever" with "only if": I was also considering that wording, but it might be too strong - also because people may be obliged to opt for distributions in case their catalogue does not support dataset series. I would suggest we discuss this during one of the DCAT calls, and keep "whenever" for the moment.

Reply:
Fine to me, let's keep as it is for the moment.